examples/dbc_io/main.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+#
+# Script testing dbc manipulation and io.
+#
+
+import os
+#import cantools
+import sys
+sys.path.append('/mnt/d/JQU/dbc/cantools')
+import cantools
+
+SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
+DBC_IO_DBC_PATH = os.path.join(SCRIPT_DIR, 'dbc_input.dbc')
+
+infile = DBC_IO_DBC_PATH
+outfile = infile.replace('_input', '_output')
+
+# Read the dbc
+db = cantools.db.load_file(infile)
+# Get a message to manipulate
+msg = db.get_message_by_name('TheMessage')

```

Comment:
Rename `msg` to `message`. I know both names exists in the code base, but I prefer `message`.